src/backends/backend/triton_backend_manager.cc

```diff
@@ -327,14 +327,20 @@ TRITONBACKEND_BackendSetState(TRITONBACKEND_Backend* backend, void* state)
 //
 // TritonBackendManager
 //
+TritonBackendManager&
+TritonBackendManager::Instance_()
```

Comment:
Singleton(). Putting _ at the end of a function name is not conforming to our coding standards.